### DISCUSSION TITLE
Is there a way to remove default headers like Accept-encoding ?

### DISCUSSION THREAD

**Discussion by VHK459** [Waboodoo/HTTP-Shortcuts#435] [Q&A]
I am sending a post request for which I want to remove some of the default headers like User-Agent, Accept encodings. Is there a way to do this?

**Comment by Waboodoo** [comment]
Currently, there is no way to remove the User-Agent or the Accept headers. The next best thing you can do is change their values by adding your own headers, but you can't get rid of them.
The only default header which you can remove is the "Connection: close" header, by enabling the "Keep connection open after request" checkbox on the "Advanced Technical Settings" screen in the shortcut editor.
